IfcCableFitting Rail Import BIMQ 2021-07-21T06:32:04.5156286Z (#152)
bSI-InfraRoom/IFC-Specification: IFC4x3/Properties/g/GroundResistance_2NdoXdKT98XeEMmk8dSBQF/Documentation.md

> The resisrance through earthing terminal exit to the ground.

Author: ```suggestion
The resistance through earthing terminal exit to the ground.
```

> The resistance through earthing terminal exit to the ground.

Reviewer: This should be rephrased. It is not clear.

> The soil or ground resistance to electrical current from the cable fitting

Reviewer: I still do not understand this. The cable fitting induces resistance in the ground/soil?
```suggestion
The soil or ground resistance to electrical current from the cable fitting.
```

> The resistance soil or ground resists electric current from the cable fitting.

Reviewer 2: ```suggestion
The soil or ground resistance to electrical current from the cable fitting
```
